src/file-4.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:17795/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:17795/project/issues?id=sample&branch=feature4&issues=AWFMrNVjD-S5K1tpKfFC&open=AWFMrNVjD-S5K1tpKfFC)

[ISSUE_KEY]: AWFMrNVjD-S5K1tpKfFC